README.md

```diff
@@ -346,17 +348,19 @@ The `outbound-edi` function can be invoked via the UI for testing.
 
 5. You can view the file using the [Buckets UI](https://www.stedi.com/app/buckets). As shown above, the output of the function includes the `bucketName` and `key` (path within the bucket) of where the generated EDI was saved.
 
-# Customizing the Bootstrap Workflow
+# Customizing the workflows
 
 The bootstrap workflow uses a sample trading partner to set up and test the read and write EDI workflows. You can customize the bootstrap workflow for your use case by doing one or all of the following:
 - [Edit the partner profile](https://www.stedi.com/docs/bootstrap/adjusting-the-workflow#add-a-trading-partner-profile) to replace the test trading partner with your real trading partners' details and requirements.
 - [Create Stedi mappings](https://www.stedi.com/docs/bootstrap/adjusting-the-workflow#map-inbound-messages). The base bootstrap repository ingests and generates JSON with a schema that closely matches EDI documents. You may need to create a mapping to transform EDI documents into a custom JSON shape for your internal system. You can also create a mapping that transforms JSON data from your system into the JSON schema required for outgoing EDI documents.
 - [Create SFTP users](https://www.stedi.com/docs/bootstrap/adjusting-the-workflow#sending-and-receiving-documents-with-sftp) for your trading partners, so they can send and retrieve EDI documents from Stedi Buckets. 
 
+You may also want to add additional Stedi products to optimize your EDI workflows. We offer free hands-on troubleshooting and customized support to get your solution into production as quickly as possible. [Contact us](https://www.stedi.com/contact) to get started.
```

Comment:
@rsioss and @joost-basic-bits are we okay with this? I want to really try to encourage folks to reach out to us because that is absolutely our preferred onboarding path for new customers...but lmk if it feels gross or overly marketing

Reply:
I don't have any major concerns with it. would be good to get @dkanter14's feedback as well

Reply:
I can live with that text, but if we really want customers to contact us, why not go all-in on that message? Something like:

> It would help us if we can set up the bootstrap repository with you. It allows us to better understand what you and other customers need. Here’s what’s in it for you:
> - Help with deploying the bootstrap workflows.
> - Help customizing the workflows to your needs.
> - Hands-on troubleshooting, should you need it.
> - All support is free of charge.
> 
> Please, [contact us](https://www.stedi.com/contact) us and we’ll set up a call.

Sure, it’s a bit pushy, but since it’s framed as ‘this is how you can help us‘, I don’t feel it’s salesy. More importantly, it’s clear: if we want customers to reach out, let’s just ask them to reach out.

Reply:
I like @joost-basic-bits aggressiveness! Let's not be ashamed of the fact that we are better off helping customers. Better off being perceived as pushy and succeeding, then being not-pushy and customers struggle!

Reply:
Thanks all! :) It feels like a bit of a stretch to expect someone to get to this README with these pretty straightforward setup instructions and then stop before they try to set up bootstrap and wait for us to do it with them. However, if we're good with being this direct, I will work in more direct language - thanks!